app/core/constant.service.js

```diff
@@ -0,0 +1,11 @@
+(function(){
+  'use strict';
+  angular.module('meganote.notes').factory('noteConstants', noteConstants);
```

Comment:
This will definitely work, but there's a quicker way to define a constant.  Instead of calling `.factory` on a module, there is actually a `.constant`.  Just give it a string for the name of the constant and the actual constant itself, then it can be injected as a dependency anywhere you need it throughout your app